src/octoprint/templates/login.jinja2

```diff
@@ -41,7 +41,12 @@
 <body>
     <div class="container">
         <form class="form-signin">
-            <h2 class="form-signin-heading" data-test-id="login-title">{{ _('Please log in') }}</h2>
+            <h2 class="form-signin-heading" data-test-id="login-title">
+                {{ _('Please log in') }}
+                <img id="logo_compact" alt="Mr Beam Logo"
```

Comment:
We might be able to append the logo directly with css on a nice way let's talk about this tomorrow